tools/build/cross-build/include/common/pwd.h

```diff
@@ -51,6 +51,7 @@ int uid_from_user(const char *name, uid_t *uid);
 const char *user_from_uid(uid_t uid, int noname);
 
 #ifdef __linux__
+#include <sys/cdefs.h>
```

Comment:
I guess this is just for the __unused? Could add a cast to void inside the function body instead?